packages/arb-avm-cpp/avm_values/src/value.cpp

```diff
@@ -268,7 +269,19 @@ struct ValuePrinter {
     std::ostream& os;
 
     std::ostream* operator()(const Buffer& b) const {
-        os << "Buffer(" << hash(b) << ")";
+        os << "Buffer(";
+        if (b.data_length() <= 64) {
+            os << "0x";
+            std::ios prev_flags(nullptr);
+            prev_flags.copyfmt(os);
+            for (auto b : b.toFlatVector()) {
+                os << std::hex << std::setw(2) << std::setfill('0') << (int)b;
+            }
+            os.copyfmt(prev_flags);
+        } else {
+            os << b.hash();
```

Comment:
I think either we should just print the data in all cases or we should change the hash output to include the word hash or something to distinguish between a 32 byte buffer and a buffer longer than 64 bytes

Reply:
Hash isn't 0x prefixed but yeah probably